packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -562,34 +607,104 @@ class RenderListWheelViewport
       perspective: _perspective,
     );
 
-    context.pushTransform(
-      // Text with TransformLayers and no cullRects currently have an issue rendering
-      // https://github.com/flutter/flutter/issues/14224.
-      false,
-      offset,
-      _centerOriginTransform(transform),
-      // Pre-transform painting function.
-      (PaintingContext context, Offset offset) {
-        context.paintChild(
-          child,
-          offset + new Offset(
-            untransformedPaintingCoordinates.dx,
-            // Paint everything in the center (e.g. angle = 0), then transform.
-            -_topScrollMarginExtent,
-          ),
-        );
-      },
-    );
+    double magnifierTopLine = size.height / 2 - _itemExtent * _magnifyRate / 2;
+    double magnifierBotLine = size.height / 2 + _itemExtent * _magnifyRate / 2;
+
+    // Some part of the child is in the center magnifier.
+    if(_magnifyRate != 1.0
+       && untransformedPaintingCoordinates.dy >= magnifierTopLine -
+                                              _itemExtent*_magnifyRate
+       && untransformedPaintingCoordinates.dy <= magnifierBotLine)
+    {
```

Comment:
{ on the same line as the closing parenthesis
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo